src/service/core/config/configmap_loader.py

```
@@ -758,19 +678,206 @@ def _validate_configs(managed_configs: Dict[str, Any]) -> List[str]:
+        for name, value in section.items():
+            if not isinstance(name, str) or not name:
+                errors.append(f'{config_key}: every entry must have a non-empty string name')
+                continue
+            try:
+                validator(name, value)
+            except pydantic.ValidationError as error:
+                errors.append(
+                    f'{config_key}.{name}: {_format_validation_error(error)}')
+            except ConfigSnapshotValidationError as error:
+                errors.append(f'{config_key}.{name}: {error}')
+            except Exception as error:  # pylint: disable=broad-exception-caught
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)